Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Systems/LoadGltfContainerSystem.cs

```diff
@@ -39,11 +44,22 @@ protected override void Update(float t)
         [None(typeof(GltfContainerComponent))]
         private void StartLoading(in Entity entity, ref PBGltfContainer sdkComponent, ref PartitionComponent partitionComponent)
         {
-            // It's not the best idea to pass Transform directly but we rely on cancellation source to cancel if the entity dies
-            var promise = Promise.Create(World, new GetGltfContainerAssetIntention(sdkComponent.Src, new CancellationTokenSource()), partitionComponent);
-            var component = new GltfContainerComponent(sdkComponent.GetVisibleMeshesCollisionMask(), sdkComponent.GetInvisibleMeshesCollisionMask(), promise);
-            component.State = LoadingState.Loading;
-            World.Add(entity, component);
+            GltfContainerComponent component;
+            if (!sceneData.TryGetHash(sdkComponent.Src, out string hash))
```

Comment:
I believe you could move these 3 lines into a single Utils method to avoid duplication

Reply:
Done!